app/src/main/res/values/strings.xml

```diff
@@ -14,13 +14,50 @@
   <string name="menu_read_aloud">Read aloud</string>
   <string name="menu_read_aloud_stop">Stop reading aloud</string>
   <string name="menu_support_kiwix">Support Kiwix</string>
+  <string name="menu_host_books">Host Books</string>
   <string name="save_media">Save Media</string>
   <string name="save_media_error">An error occurred when trying to save the media!</string>
   <string name="save_media_saved">Saved media as %s to Android/media/org.kiwix…/</string>
   <string name="rescan_fs_warning">Scanning for downloadable content, please wait…</string>
   <string name="search_label">Search</string>
   <string name="choose_file">Select a Content File (*.zim)</string>
   <string name="open_in_new_tab">Open link in new tab?</string>
+  <string name="error_nozimfilesfound">No content found on your device.\nTake a look at the Help Page to get directions on how to load content into Kiwix.\nIf you did put a ZIM file on your device/external storage, you might retry in a minute or restart your device.</string>
```

Comment:
not snake case

Reply:
@macgills This line was not added by me, i re-ordered it mistakenly. I just looked into this issue and found out that there are tons of **faulty snake cases** like this in the `strings.xml` and would require a separate **PR** for fix.

Reply:
Why do these appear as new additions then? If you merged from the source branch in then these should not show up as new in this PR

Reply:
Idk what exactly happened.